lib/ui/views/homepage/widgets/favourite_widget.dart

```diff
@@ -9,19 +9,23 @@ class FavoriteWidget extends StatelessWidget {
   const FavoriteWidget({
     super.key,
     required this.productModel,
+    this.usePadding = true,
+    this.color,
   });
 
   final ProductModel productModel;
```

Comment:
remember my comment above about changing this to: `final ProductModel product`

Reply:
Yessur!🫡